sensor_msgs/Image rgb
sensor_msgs/Image depth
---
float32[]   label_distribution
